lib/todo/command_handler.ex

```diff
@@ -1,35 +1,27 @@
 defmodule ToDo.CommandHandler do
-  import ToDo.ConsoleDisplay
+  import ToDo.AddTasksCommand
+  import ToDo.DeleteTaskCommand
+  import ToDo.EditTaskCommand
```

Comment:
Generally, try and avoid using `import`. The reason this not very idiomatic is because if for example I'm looking down at `delete_task(task_number, previous_state)`, I don't know from which module it's coming from so instead it's better not to use `import` and reference the module directly where it's necessary: `ToDo.DeleteTaskCommand.delete_task`.